app/views/shared/_attributes.html.erb

```diff
@@ -1,37 +0,0 @@
-      <table class="table">
-        <tr>
-          <th><span class="attribute-label h4">Title:</span></th>
-          <td><%= work.title %></td>
-        </tr>
-        <tr>
-          <th><span class="attribute-label h4">Owner:</span></th>
-          <td><%= link_to_profile work.depositor("no depositor value") %></td>
-        </tr>
-        <tr>
-          <th><span class="attribute-label h4">Depositor:</span></th>
-          <td>
-            <% if work.proxy_depositor  %>
```

Comment:
Presently it's just displaying proxy_depositor as Depositor if it exists.  If it doesn't exist, it only displays the Owner field. I think that makes sense. Why do we want to see the depositor if it's the same as the owner?